### PR TITLE
Clean up unused non-CMR colors

### PR DESCRIPTION
**Pull request by tiffwong**
For some reason, it's not showing the files I changed unless you click on my [initial commit](https://github.com/linode/manager/pull/7278/commits/ed34b451bdfee7122dd596d896b208f1c34e3449) 💁‍♀️

### CODE REVIEW

File: packages/manager/src/themeFactory.ts [hunk 1]
```diff
@@ -318,8 +316,6 @@ const themeDefaults: ThemeDefaults = ({ spacingOverride: spacingUnit }) => {
       grey4: '#8C929D',
       grey5: '#f5f5f5',
       grey6: '#e3e5e8',
```
Comment (Jskobos) [inline packages/manager/src/themeFactory.ts]: Weird that there's a grey9 now but not a grey7 or 8

**Review by Jskobos — COMMENTED** [review]
I think this is probably because these changes were made in other PRs as well so merging develop just turns this branch into develop. I spot checked a few of the changes and they're all present in develop.

**Comment by Jskobos** [comment]
@tiffwong is there anything in here that didn't get changed separately? If not we can close this out.

**Comment by tiffwong** [comment]
Closing since changes are present in develop

**Pull request closed without merging**